MAINTAINERS: add @ivanhercaz to past organization members (#5975)
tldr-pages/tldr: MAINTAINERS.md

- Leandro Ostera ([@ostera](https://github.com/ostera)):
  until [18 January 2018](https://github.com/tldr-pages/tldr/issues/1878#issuecomment-358610454)
- Waldir Pimenta ([@waldyrious](https://github.com/waldyrious)): until [26 August 2018](https://github.com/tldr-pages/tldr/issues/2257)

## Past organization members

> Organization members who have been inactive for some time, but still have write access to all the repositories in the tldr-pages organization and are welcome back any time to return to their old role.

Reviewer: ```suggestion
Organization members who have been inactive for some time, still have write access to all the repositories in the tldr-pages organization, after losing organization status, previous members are welcome back any time to return to their old role.
```

Reviewer 2: Let's reserve this type of change for a different PR.  I have a PR #5770 that can touch this file as well.  (Also, `losing` is really too negative, I think.)

Reviewer: ```suggestion
Organization members who have been inactive for some time, still have write access to all the repositories in the tldr-pages organization. After releasing organization status, previous members are welcome back any time to return to their old role.
```

Reviewer 2: How about just "removing"?

Reviewer: releasing is less _negative_ than removing

Reviewer 2: True, so much so that it's too positive.  "Remove" is very neutral and implies nothing.  "Lose" implies that it's a good thing to have and its removal is not desired.  "Release" implies that its nothing but a burden and an unwanted one, at that.

Reviewer: @marchersimon I think this is the most I have needed your opinion!

Reviewer 3: Do we need a dedicated section like this at all? I thought that the dates next to people's names was enough.

Author: That's how it was described in [COMMUNITY-ROLES](https://github.com/tldr-pages/tldr/blob/main/COMMUNITY-ROLES.md#removing-inactive-organization-members).

> Open a PR moving their name to the "Past organization owners" section in MAINTAINERS.md.

Author: So we should put both past organization members and owners to the section `Past organization owners`?

Reviewer 3: Oh oops! I've got confused. I'm not sure whether we'd want separate sections or not? Maybe 1 as `Past organization owners`?

Reviewer 3: Let's go with separate sections for now. We can always change it later.

Reviewer 4: @marchersimon could you resolve the conflicts?
